core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -1070,10 +1084,25 @@ class SocketServerTest extends JUnitSuite {
     }
   }
 
+  private def withTestableControlPlaneServer(config : KafkaConfig, testWithServer: TestableSocketServer => Unit) = {
```

Comment:
Instead of creating a new method, could we just make withTestableServer have an optional KafkaConfig param?

Reply:
Agreed